Introducing the new Log Purchases Action (#2730)
segmentio/segment-docs: src/connections/destinations/catalog/actions-amplitude/index.md

Once you have a mapping, you can follow the steps in the Destinations Actions documentation on [Customizing mappings](/docs/connections/destinations/actions/#customizing-mappings).

### Log Purchases in existing destination instances

Initially the Log Event Action was reporting purchases to Amplitude for all events containing a `products` array, even if the products were just added to cart. This inflated the LTV Chart in Amplitude.

To resolve this, purchase reporting takes place in a new Action called Log Purchase.

> Any instances created after x/y has both Log Event and Log Purchase configured, but for prior instances you need to manually add the Log Purchases Action to report purchases to Amplitude:
> 
> 1. Add a new Mapping for the Log Purchases Action. The default trigger for this action is Order Completed events.
> 2. Modify the Trigger if you need to report purchases for any other events
> 3. Modify the Trigger of Log Event to exclude these same events, to avoid sending the same event twice
> 4. Enable the Log Purchases mapping

Reviewer: ```suggestion
Instances created after x/y have both Log Event and Log Purchase configured.  For instances created prior to x/y, you need to manually add the Log Purchases Action to report purchases to Amplitude. 

To manually add the Log Purchases Action: 
1. Add a new Mapping for the Log Purchases Action. The default trigger for this action is Order Completed events.
2. Modify the Trigger if you need to report purchases for any other events.
3. Modify the Trigger of Log Event to exclude these same events. This helps you to avoid sending the same event twice. 
4. Enable the Log Purchases mapping. 
```

Reviewer: @nielst I have the same question as Mark. Is `x/y` supposed to represent a date? If so, what date is it?

Author: thanks for your help here! yup its a date, should be in a week or so

Once you have a mapping, you can follow the steps in the Destinations Actions documentation on [Customizing mappings](/docs/connections/destinations/actions/#customizing-mappings).

### Log Purchases in existing destination instances

Initially, the Log Event Action was reporting purchases to Amplitude for all events containing a `products` array, even if the products were just added to cart. This inflated the LTV Chart in Amplitude.

To resolve this, purchase reporting takes place in a new Action called Log Purchase.

> Instances created after x/y have both Log Event and Log Purchase configured.  For instances created prior to x/y, you need to manually add the Log Purchases Action to report purchases to Amplitude. 

Reviewer 2: ```suggestion
For instances created prior to before the Log Purchases action was released, you need to manually add the Log Purchases Action to report purchases to Amplitude. 
```
